internal/operators/ocs/validations.go

```diff
@@ -0,0 +1,120 @@
+package ocs
+
+import (
+	"encoding/json"
+	"math"
+
+	"github.com/docker/go-units"
+	"github.com/openshift/assisted-service/internal/host"
+	"github.com/openshift/assisted-service/models"
+	"github.com/sirupsen/logrus"
+)
+
+type OCSValidator struct {
+	OCSMinimumCPUCount        int64 `envconfig:"OCS_MINIMUM_CPU_COUNT" default:"24"`
+	OCSMinimumRAMGB           int64 `envconfig:"OCS_MINIMUM_RAM_GB" default:"66"`
+	OCSRequiredDisk           int64 `envconfig:"OCS_MINIMUM_DISK" default:"3"`
+	OCSRequiredHosts          int   `envconfig:"OCS_MINIMUM_HOST" default:"3"`
+	OCSRequiredCPUCount       int64 `envconfig:"OCS_REQUIRED_CPU_COUNT" default:"30"`
+	OCSRequiredRAMGB          int64 `envconfig:"OCS_REQUIRED_RAM_GB" default:"72"`
+	OCSRequiredMasterCPUCount int64 `envconfig:"OCS_REQUIRED_MASTER_CPU_COUNT" default:"16"`
+	OCSRequiredMasterRAMGB    int64 `envconfig:"OCS_REQUIRED_MASTER_RAM_GB" default:"64"`
+	OCSMasterWorkerHosts      int   `envconfig:"OCS_REQUIRED_MASTER_WORKER_HOSTS" default:"5"`
+	OCSMinimalDeployment      bool  `envconfig:"OCS_MINIMAL_DEPLOYMENT" default:"false"`
+	OCSStorageCapacity        int64 `envconfig:"OCS_STORAGE_CAPACITY" default:"10"` // to claim for OSDs
+	OCSDisksAvailable         int64 `envconfig:"OCS_DISKS_AVAILABLE" default:"3"`
+}
+
+// ValidateOCSRequirements is used to validate min requirements of OCS
+func (o *OCSValidator) ValidateOCSRequirements(log logrus.FieldLogger, cluster *models.Cluster, h host.API) bool {
```

Comment:
Consider creating an interface out of this package then you will not need to run actual code bug a mock in other packages.
We are using mockgen to generate mocks.